common/element_handle.go

```diff
@@ -915,7 +915,7 @@ func (h *ElementHandle) InputValue(opts goja.Value) string {
 func (h *ElementHandle) IsChecked() bool {
 	result, err := h.isChecked(h.ctx, 0)
 	if err != nil && err != ErrTimedOut { // We don't care anout timeout errors here!
-		k6ext.Panic(h.ctx, "element isChecked: %w", err)
+		k6ext.Panic(h.ctx, "getting element is checked: %w", err)
```

Comment:
`"getting element is checked"` sounds awkward. Same for the other `ElementHandle` errors.

Maybe:

```suggestion
		k6ext.Panic(h.ctx, "getting element checked value: %w", err)
```